degirum_tools/notifier.py

```diff
@@ -32,7 +34,13 @@ def __init__(
         *,
         message: str = "",
         holdoff: Union[Tuple[float, str], int, float] = 0.0,
-        token: Optional[str] = None,
+        # For now, token contains path to yaml. Eventually, it'll be the token and we can fetch the yaml from 
+        # the backend using this token as an identifier.

```

Comment:
Usually, these arg comments are provided in docstring just below

Reply:
Got it. Then should my explanation of tags and whatnot also go in the doc string?